packages/ui/package.json

```diff
@@ -102,15 +103,15 @@
 		"eslint-plugin-react": "^7.32.2",
 		"eslint-plugin-react-hooks": "^4.6.0",
 		"eslint-plugin-simple-import-sort": "^7.0.0",
-		"eslint-plugin-storybook": "^0.6.12",
+		"eslint-plugin-storybook": "^0.8.0",
 		"eslint-plugin-tailwindcss": "^3.6.1",
 		"eslint-plugin-unused-imports": "^1.1.2",
 		"fs-extra": "^11.1.0",
 		"globby": "^13.1.3",
 		"npm-run-all": "^4.1.5",
 		"prop-types": "^15.8.1",
 		"react-dom": "^18.2.0",
-		"storybook": "^8.0.4",
+		"storybook": "^8.2.6",
```

Comment:
Some of the Storybook config changes came from running the auto upgrade task from the Storybook CLI.